pkg/interfaces/contracts/vault/IVaultAdmin.sol

```diff
@@ -194,15 +197,14 @@ interface IVaultAdmin {
     function unpauseVaultBuffers() external;
 
     /**
-     * @notice Adds liquidity to an yield-bearing token buffer (linear pool embedded in the vault).
-     *
+     * @notice Adds liquidity to an yield-bearing buffer (one of the Vault's internal ERC4626 buffers).
      * @param wrappedToken Address of the wrapped token that implements IERC4626
      * @param amountUnderlyingRaw Amount of underlying tokens that will be deposited into the buffer
      * @param amountWrappedRaw Amount of wrapped tokens that will be deposited into the buffer
-     * @param sharesOwner Address of contract that will own the deposited liquidity. Only
-     *        this contract will be able to remove liquidity from the buffer
-     * @return issuedShares the amount of tokens sharesOwner has in the buffer, expressed in underlying token amounts
-     *         (it is the BPT of the vault's internal linear pools)
+     * @param sharesOwner Address of contract that will own the deposited liquidity. Only this contract will be able
```

Comment:
```suggestion
     * @param sharesOwner Address of EOA that will own the deposited liquidity. Only this EOA will be able
```

I realized that it can be a contract or a wallet, right? Should we call it EOA (or wallet, or something else)?

Reply:
Maybe just "address". It could be either one; certainly not constrained to be a contract.